packages/fxa-admin-panel/scripts/test-ci.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash -ex
 
-yarn workspaces focus fxa-admin-panel
```

Comment:
Focus runs an install again, which runs a linking step which is expensive. This isn't necessary.